starter-code/functions-and-arrays.js

```diff
@@ -63,7 +84,12 @@ console.log(averageLength);
 
 // Unique Arrays
 function uniquifyArray (array) {
-
+  array.forEach(function(e,i){
+    while(array.indexOf(e,i+1)!=-1){
+      array.splice(array.indexOf(e,i+1));
+    }
+  });
+ return array;
```

Comment:
Pobre "communion" qué os ha hecho para que no le dejéis salir en el uniqueArray?? :(